src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -49,6 +49,7 @@ const ROWS_AMOUNT = 4
 export const ChannelView: React.FC = () => {
   const [openUnfollowDialog, closeUnfollowDialog] = useDialog()
   const { id } = useParams()
+  const [searchParams, setSearchParams] = useSearchParams({ tab: [...TABS] })
```

Comment:
What is this parameter supposed to do? Why are we passing an array?

Reply:
I was trying something but I noticed it isn't actually needed so I'm removing it